docs/zh/operation/pod_schedule_optimization.md

```diff
@@ -6,7 +6,8 @@
 
 1.支持K8s原生调度器,以及Volcano, Yunikorn等实现Pod数据亲和性调度  
 2.将Pod优先调度到有数据缓存能力的节点  
-3.当Pod不使用数据集时，可以尽量避免调度到有缓存的节点
+3.可以通过指定Pod Label的形式，将Pod强制调度到有数据缓存的节点

```

Comment:
Could you also update the english version as well?

Reply:
ok, I will update it.